scripts/spectrum-vars.js

```diff
@@ -230,7 +230,7 @@ spectrumPaths.forEach(async (spectrumPath, i) => {
                 '..',
                 'tools',
                 ...packageDir,
-                `theme-${theme}.css`
+                `spectrum-theme-${theme}.css`
```

Comment:
Create the Spectrum specific version of this file and then point to it with our entry point where we've added the `color-scheme` information.